.github/workflows/ci.yml

```diff
@@ -37,6 +37,8 @@ jobs:
       with:
         node-version: ${{ matrix.node }}
         cache: 'pnpm'
+    - name: Install npm@7
+      run: node - add --global npm@7
```

Comment:
How would this work? Node is not a package manager.
It should be

```
run: pnpm add --global npm@7
```

Reply:
sry i mistaked it... updated now